odl/operator/tensor_ops.py

```diff
@@ -866,8 +868,14 @@ def _call(self, x, out=None):
                 # sparse matrices
                 out[:] = self.matrix.dot(x)
             else:
-                with writable_array(out) as out_arr:
-                    self.matrix.dot(x, out=out_arr)
+                if (parse_version(np.__version__) < parse_version('1.13.0') and
+                        x is out):
+                    # Workaround for bug in Numpy < 1.13 with aliased in and
```

Comment:
wait, do we even allow np>=1.13 atm? or is this future profing?

Reply:
No, not currently. But since the last `conda` update (an ASTRA update) it has become notoriously hard to install a lower version, also since they forgot to add Numpy 1.12 with nomkl. 

So it's easier than ever to install the wrong version of Numpy. That will break some of the tests, but at least there are no extra bugs. And the aliased version gives a speedup that would be nice to have when not broken.

Reply:
Ok, how much is left until we have tensors in and can handle this "natively" so to say?

Reply:
There's one bug with the Huber functional that breaks the large-scale test, and it's not on master. Need to check. Then there is the breakage of the `tensorflow` contrib code which I imagine is not unimportant :-). You know what to change there I guess, I only did the obvious stuff (`size` -> `shape` roughly). Then, I'll check again the examples. Other than that, it's ready I think.

Reply:
Well getting this fixed before the course would be great so we have better TF interop for then, I suggest we try to rush that and get it somewhat battle-tested before the 12:th next month